rllib/algorithms/algorithm_config.py

```diff
@@ -230,8 +230,8 @@ def overrides(cls, **kwargs):
 
         Raises:
             KeyError: In case a non-existing property name (kwargs key) is being
-            passed in. Valid property names are taken from a default AlgorithmConfig
-            object of `cls`.
+                passed in. Valid property names are taken from a default
+                AlgorithmConfig object of `cls`.
```

Comment:
why are these indentation changes also required?

Reply:
With googledoc docstrings, multiline parameter descriptions need to have a hanging indent, otherwise sphinx tries to interpret each line as a new parameter (which causes broken references)